psydac/cad/geometry.py

```diff
@@ -23,7 +23,8 @@
 from psydac.fem.tensor       import TensorFemSpace
 from psydac.mapping.discrete import SplineMapping, NurbsMapping
 
-from sympde.topology import Domain, Line, Square, Cube
+from sympde.topology import Domain, Line, Square, Cube, NCubeInterior
```

Comment:
Do we still use classes `Line`, `Square` and `Cube` in this file? If not, please remove their import

Reply:
yes we still them they are used in the file